tests/database/test_chaindb.py

```diff
@@ -62,21 +62,21 @@ def block(request, header):
 
 def test_add_block_number_to_hash_lookup(chaindb, block):
     block_number_to_hash_key = make_block_number_to_hash_lookup_key(block.number)
-    assert not chaindb.exists(block_number_to_hash_key)
-    chaindb.add_block_number_to_hash_lookup(block.header)
-    assert chaindb.exists(block_number_to_hash_key)
+    assert not chaindb.db.exists(block_number_to_hash_key)
```

Comment:
As far as I can tell, this change isn't necessary, because chaindb still has the `exists` proxy. Why the change?

Should we encourage callers to call `exists` one way or the other? My instinct is to suggest that they not reach inside the underlying db like this, but it's a soft opinion.

Reply:
Leftover from me thinking I could remove the `exists()` method.

Reply:
Changed back in f75c376